### DISCUSSION TITLE
Website pages layout is messy

### DISCUSSION THREAD

**Discussion by UE2lab** [community/community#84188] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi everyone!

I am here to find answers related to my messy website layout. I have my website page description is messy and has different margin sizes for each content. Now, i am still finding a way to solve it but i haven't got any. So, I hope I can get the answers from you guys! Thank you !!

![image](https://github.com/community/community/assets/124585923/4ee071ad-5c93-442a-92dd-77023e872825)

it is supposed to be centered and having same indent for every content like this:
![image](https://github.com/community/community/assets/124585923/dd088bcd-caa5-46bb-b6f7-5581d19df354)

**Comment by ananuness** [comment]
Okay, so first, it would be nice if you showed the code structure, but a good solution is:

1- See where there are margins and remove them to do it from scratch without having conflicts.
2- Enclose all content to be centered by a div, or even the main tag. And in this tag you would apply the margins.
3- As you just want to centralize, you can use:
```css
/* 0px top and bottom, auto both sides to centralize */
margin: 0 auto;
```

I hope it helps!

**Reply by UE2lab** [reply]
Thank you for the reply. However, I still failed to manage it. Here I am providing the code structure the main css https://github.com/UE2lab/UE2lab/blob/gh-pages/css/main.scss and the HTML page https://github.com/UE2lab/UE2lab/blob/gh-pages/_layouts/research.html

Do you know if there is anything related here to fix my problem? 

Thanks!
